core/server/data/migrations/versions/3.23/02-remove-bulk-email-settings.js

```diff
@@ -0,0 +1,48 @@
+const logging = require('../../../../../shared/logging');
```

Comment:
@naz @rishabhgrg @daniellockyer Could you please review this migration? :relaxed: